modules/end-user-portal/src/Core/ServiceCollectionExtensions.cs

```diff
@@ -65,8 +65,8 @@ public static IServiceCollection ConfigureApplicationServices(this IServiceColle
         services.AddAntiforgery(options =>
         {
             options.SuppressXFrameOptionsHeader = true;
-            options.Cookie.HttpOnly = false;
-            options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
+            options.Cookie.HttpOnly = true;
+            options.Cookie.SecurePolicy = CookieSecurePolicy.SameAsRequest;
             options.Cookie.SameSite = SameSiteMode.None;
         });
 
```

Comment:
In the code block where we have AddHsts do we not need a if environment == Development?

Reply:
I'm not sure how this interacts with the traffic outside the cluster being in front of the ALB and thus covered by TLS.

Reply:
This will enforce that any form submission matches the scheme that the request was made in.

Given that the LB will be handling the SSL termination and forwarding this should be ok.